test/local/bounce_tests.js

```diff
@@ -0,0 +1,282 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var EventEmitter = require('events').EventEmitter
+var sinon = require('sinon')
+var test = require('../ptaptest')
+var error = require('../../lib/error')
+var P = require('../../lib/promise')
+
+var mockBounceQueue = new EventEmitter()
+mockBounceQueue.start = function start() {}
+
+var mockLog = {
```

Comment:
Noticed that mockLog is reused in other test files. Is this something we could put in common area? Doesn't have to be fixed in this PR but maybe an issue for it?